templates/etc/haproxy/global.cfg.j2

```diff
@@ -55,6 +55,10 @@
   ssl-default-bind-ciphers {{ haproxy_global_ssl_default_bind_ciphers }}
 {% endif %}
 
+{% if haproxy_global_ssl_default_bind_ciphersuites != false and haproxy_global_ssl_default_bind_ciphersuites | length > 0 %}
```

Comment:
Can you change the check to:
```
{% if haproxy_global_ssl_default_bind_ciphersuites %}
```
? 

Same below

Reply:
Fixed in https://github.com/Oefenweb/ansible-haproxy/pull/99/commits/b95a3658e956515e42e11379d3bccd54a884a45a.